packages/create-plugin/src/utils/utils.git.ts

```diff
@@ -0,0 +1,37 @@
+import { exec as nodeExec } from 'node:child_process';
+import { promisify } from 'node:util';
+
+const exec = promisify(nodeExec);
+
+export async function isGitDirectory() {
+  let command = 'git rev-parse --is-inside-work-tree';
+
+  try {
+    const response = await exec(command);
+
+    return response.stdout.trim() === 'true';
+  } catch (error) {
+    if (error instanceof Error && error.message.includes('fatal: not a git repository')) {
+      return false;
+    }
+
+    throw new Error(
```

Comment:
I prefer we return false in any case here:

We must take in consideration that some people use their OS in a language different than English and we won't detect those cases.

even if all the messages were always in English we would be obscuring the real error by throwing our own error instead of the original git one.

Reply:
That's a good input, thanks 👍 Will change it accordingly.

Reply:
Fixed in https://github.com/grafana/plugin-tools/pull/702/commits/907a1e2123438cab41adf6108a82d9d853640fe7.